src/components/PersonRow/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const [searchParams] = useSearchParams();
+
+  return (
+    <td>
+      <Link
+        to={{
+          pathname: `./${person.slug}`,
+          search: searchParams.toString(),
+        }}
+        className={classNames({ 'has-text-danger': person.sex === 'f' })}
```

Comment:
You already have the SexFilter component, so there's no need to use the 'f' anymore.